src/styles/blocks/catalog.scss

```diff
@@ -17,6 +19,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $duration ease-in-out, color $duration ease-in-out;
```

Comment:
Need separate variable duration of transform duration for transform and color.

Reply:
No, you don't need a separate duration!